templates/bot/ts/notification-trigger-http/src/httpTrigger.ts

```diff
@@ -29,52 +29,52 @@ const httpTrigger: AzureFunction = async function (
         description: `This is a sample http-triggered notification to ${target.type}`,
         notificationUrl: "https://www.adaptivecards.io/",
       })
+    );
 
-      // Note - you can filter the installations if you don't want to send the event to every installation.
+    // Note - you can filter the installations if you don't want to send the event to every installation.
 
-      /** For example, if the current target is a "Group" this means that the notification application is
-       *  installed in a Group Chat.
-      if (target.type === "Group") {
-        // You can send the Adaptive Card to the Group Chat
-        await target.sendAdaptiveCard(...);
+    /** For example, if the current target is a "Group" this means that the notification application is
+     *  installed in a Group Chat.
+    if (target.type === NotificationTargetType.Group) {
+      // You can send the Adaptive Card to the Group Chat
+      await target.sendAdaptiveCard(...);
 
-        // Or you can list all members in the Group Chat and send the Adaptive Card to each Team member
-        const members = await target.members();
-        for (const member of members) {
-          // You can even filter the members and only send the Adaptive Card to members that fit a criteria
-          await member.sendAdaptiveCard(...);
-        }
+      // Or you can list all members in the Group Chat and send the Adaptive Card to each Team member
+      const members = await target.members();
+      for (const member of members) {
+        // You can even filter the members and only send the Adaptive Card to members that fit a criteria
+        await member.sendAdaptiveCard(...);
       }
-      **/
-
-      /** If the current target is "Channel" this means that the notification application is installed
-       *  in a Team.
-      if (target.type === "Channel") {
-        // If you send an Adaptive Card to the Team (the target), it sends it to the `General` channel of the Team
-        await target.sendAdaptiveCard(...);
+    }
+    **/
 
-        // Alternatively, you can list all channels in the Team and send the Adaptive Card to each channel
-        const channels = await target.channels();
-        for (const channel of channels) {
-          await channel.sendAdaptiveCard(...);
-        }
+    /** If the current target is "Channel" this means that the notification application is installed
+     *  in a Team.
+    if (target.type === NotificationTargetType.Channel) {
+      // If you send an Adaptive Card to the Team (the target), it sends it to the `General` channel of the Team
```

Comment:
"it sends it" seems weird. How about "it will be sent"?

Reply:
Make sense, I'll create another PR for wording fix since this one is already merged...